packages/cashier/src/components/crypto-fiat-converter/timer.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { Text, useInterval } from '@deriv/components';
+
+type TTimerProps = {
+    onComplete: VoidFunction;
+};
+
+const Timer = ({ onComplete }: TTimerProps) => {
+    const initial_time = 60;
+    const [remaining_time, setRemainingTime] = React.useState<number>(initial_time);
+
+    useInterval(() => {
```

Comment:
I feel bad about this :smile: 

This will run every second and we are not clearing this anywhere, not sure but it may cause some performance issue :thinking: 

Probably settimeout would be a better choice since it will run once, after a second :thinking: 

cc: @farzin-deriv @balakrishna-deriv

Reply:
since this comment is out of the scope of this card you can ignore it :pray:

Reply:
@niloofar-deriv sorry I missed this comment. yes this is out of this card's scope. I just moved this component to an external file.